crates/wysiwyg/src/composer_model/delete_text.rs

```diff
@@ -320,4 +324,18 @@ mod test {
         assert_eq!(*handles[2].raw(), vec![0, 9, 2]);
         assert_eq!(handles.len(), 3);
     }
+
+    #[test]
+    fn delete_nodes_refuses_to_delete_root() {
+        let mut model = cm("|");
+        model.delete_nodes(vec![model.state.dom.document_handle()]);
+        assert_eq!(tx(&model), "|")
+    }
+
+    #[test]
+    fn delete_nodes_refuses_recursively_to_delete_root() {
+        let mut model = cm("a|");
+        model.delete_nodes(vec![DomHandle::from_raw(vec![0])]);
+        assert_eq!(tx(&model), "")
```

Comment:
Will this test clash with the "always show cursor" one?

Reply:
The always show cursor one builds on top and fixes this.